pkg/controller/node/nodecontroller.go

```diff
@@ -371,6 +402,42 @@ func (nc *NodeController) Run() {
 			}
 		}, nc.nodeMonitorPeriod, wait.NeverStop)
 
+		go wait.Until(func() {
+			nc.evictorLock.Lock()
```

Comment:
@gmarek 
> Once you do that you can slightly change the semantics of values in the queues - value.Value can hold PodName instead of NodeName. NodeUID won't be needed anymore, so value.UID (your message here) could be a just a PodNamespace string. WDYT?

+1 for removing NodeUID here, but value.Value can't hold only PodName, since there may be pods with same name in different namespaces.

What about holding "podNamespace/podName"(we can call `cache.MetaNamespaceKeyFunc(pod)` to get a key in this format) instead?  For value.UID, could be just pod.UID. And then we can also omit the definition of evictionMessage.